src/rules/top-level-interop.js

```diff
@@ -24,8 +24,10 @@ const rule = {
 		type: "problem",
 
 		docs: {
+			recommended: true,
```

Comment:
```suggestion
			recommended: false,
```

Same as the previous comment.